### ISSUE TITLE
Staging Review finding: Instructions may be missed by screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#84586]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Contention Classification
**Product:** Disability Comp Application (21-526 EZ)
**Feature:** Improve conditions submission page

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Screen reader users navigate a page a variety of ways depending on context. Sometimes they navigate by heading, sometimes they navigate by ARIA landmark, and sometimes they just press [tab] to work their way through all of the interactive elements on the page.

In a form workflow --- especially several pages deep into a multi-page flow --- the typical behavior we'd expect is to [tab] through the interactive elements of the page, working their way through the form as efficiently as possible. But that means their screen reader is likely to only announce to them the text that's programmatically associated with each interactive element (links, buttons, form inputs). Anything not programmatically associated is likely to be missed.

As currently coded, all of the instructions on the page ("Tell us the new conditions...," "What if my condition...," and "Examples of conditions" sections) may be skipped by a screen reader user working their way through the page as efficiently as possible.

I don't think the instructions are strictly *necessary* to complete the task. But they do provide important context, and the "What if my condition isn't listed" section is important information to clarify how to handle that situation. The good heading structure helps here too, since I would expect most screen reader users to turn to that navigation pattern if they want to check to see if there's more content on the page.

But even if this isn't critical, there's a major benefit in ensuring that screen reader users have an opportunity to hear all of the instructions before they enter their conditions.

## Recommended action

Recommendation here is pretty straightforward! Wrap the whole thing in a `fieldset` (the instructions and the cards), and then put all of the intro text inside the `legend` for that `fieldset`. You'll need to do some extra CSS work to style the headings, paragraphs, and unordered list appropriately, but that's all that's required.

Once the `fieldset` and `legend` are in place, the first time a screen reader user tabs to an interactive element in the `fieldset` (either the "Enter your condition" input or an Edit button) their screen reader will announce the instruction text.

Please feel free to reach out to me via Slack if you have any questions about this!

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by ashtondragon** [comment]
Resolved during this collab cycle https://github.com/department-of-veterans-affairs/va.gov-team/issues/105226
